### ISSUE TITLE
Forced lightning install

### ISSUE THREAD

**Issue by carmocca** [Lightning-AI/lightning-Habana#26]
## 🐛 Bug

`pytorch_lightning` users who install this package, get `lightning` installed automatically

### To Reproduce

```python
pip install pytorch_lightning
pip install lightning-habana
pip list | grep lightning
```

### Expected behavior

lightning habana should use the existing lightning implementation and not install one
This will mean that it requires that lightning is installed first.

cc @Borda

**Comment by Borda** [comment]
> lightning habana should use the existing lightning implementation and not install one

It does but not PL 

> This will mean that it requires that lightning is installed first.

Not really, once installed, it is used...

**Comment by carmocca** [comment]
Consider the case where the user has only installed `pytorch_lightning` and is using `pytorch_lightning` imports

Since installing `lightning-habana` will force install `lightning`: https://github.com/Lightning-AI/lightning-Habana/blob/main/requirements.txt#LL1C1-L1C1
This statement will trigger https://github.com/Lightning-AI/lightning-Habana/blob/a79d6d77e79e626acb14e78dbc5c96ad1712bdd9/src/lightning_habana/pytorch/strategies/parallel.py#L21 and the package will be unusable because it will return a `lightning` strategy for a `pytorch_lightning` trainer.

My proposal is that `lightning` is removed from this package's requirements

**Comment by Borda** [comment]
ok, the limitation here is that if we remove it from requirements while installing pip won't be checking any compatibility among installed and eventually additional packages... so we can do you to resolve this issue bit still not an ideal way...

**Comment by carmocca** [comment]
Why would we need this compatibility check by pip? At runtime, you can check whether the lightning version is valid.

**Comment by Borda** [comment]
This shall be also resolved before the final 1.0 release
cc: @jerome-habana
